### ISSUE TITLE
install script should not wipe host machine's crontab

### ISSUE THREAD

**Issue by geota** [Ubiquiti-App/UNMS#83]
This isn't a bug, but a feature request.

Honestly, the install script likely could be replaced with just the docker-compose.yml without all the templating and a short README on which env variables to set. IMHO right now it is too intrusive to the host machine. I would cut a pull request, but it looks like this repo just contains release artifacts.

Current behavior:
   - install script currently overwrites the host machines user's 'unms' crontab. this caught me off guard and I had to re-create my crontab because i modified the script to install under my existing docker user. the script first creates the unms user on the host machine to use. for my own reasons, i wanted it run under my existing docker user.

Expected behavior:
   - the install script should ideally should be replaced with a docker-compose.yml and a README
   -  if that is not feasible, the install script should prompt for user to run the stack under
   - the install script should append to current configured user's crontab and only add the entry if it doesn't exist

```
  if crontab -l -u "${USERNAME}"; then
    if ! crontab -u "${USERNAME}" -r; then
      echo "Failed to clean crontab"
      exit 1
    fi
  fi
```

**Comment by geota** [comment]
this isn't a bug, but more a request... it doesnt wipe the user's crontab, but the crontab of the 'unms' user it creates. 

ide be happy to try to contribute a working docker-compose that is working well on my archlinux distro

**Comment by woopstar** [comment]
Should be fixed by #103 request

**Comment by jflidr** [comment]
Fixed in 0.11.1.
